doc/source/contributing/issue_reports.md

```diff
@@ -29,10 +29,8 @@ If relevant, also include the complete error messages and tracebacks.
 To avoid cluttering the issue body, you can optionally use the `details`
 html tag in GitHub to create a simple dropdown, like so:
 
-````
 ```
 <details><summary>Click to see full traceback</summary>
 long and barely comprehensible error traceback
-```
 </details>
-````
+```
```

Comment:
This was intented to provide a copy pasteable thing for issue tracebacks. I think it would be best to use tabs to show side by side how it looks and the code that generates it. 

It was broken, but if it had been 

`````
````
<details><summary>Click to see full traceback</summary>

```
long and barely comprehensible error traceback
```

</details>
````
`````

it would then have rendered correctly so that the traceback is formatted as code. If we do these changes we are recommending users to share tracebacks formatted as text instead

Reply:
Done